src/Components/Dropdowns/Composed/TypeAheadDropDown.tsx

```diff
@@ -19,7 +12,8 @@ import {
 
 import {Input} from '../../Inputs/Input'
 import {DropdownHeader} from '../DropdownHeader'
-
+import {FixedSizeList as List} from 'react-window'
```

Comment:
Is there a reason for using an alias here (List instead of FixedSizeList)?

Reply:
I can't seem to remember my justification 😅 I'll change it back to FixedSizeList. 👍